### DISCUSSION TITLE
selenium docker tags

### DISCUSSION THREAD

**Discussion by stocksr** [renovatebot/renovate#18084] [Request Help]
Hi We have just started using Selenium at work and renovate has started proposing updates to their docker images 
however as per   https://github.com/SeleniumHQ/docker-selenium/wiki/Tagging-Convention

some of the images get multiple tags.

one is tagged with the Selenium product version e.g. 4.0.0 from the example and one is tagged starting with the browser version embedded in the image eg. from the example  firefox  78.0.2  

is there any way to tell renovate to update all the images from a mono repo to the same version, and ignore higher versions when only one image from the monorepo has a higher version.

**Comment by rarkins** [comment]
Meaning if you're on 4.x you don't want >=78?

You should be able to achieve this by using packageRules with matchCurrentVersion and allowedVersions
